bridgebot/deck.py

```diff
@@ -34,17 +48,17 @@ def __deal(cards):
             The bridge hands to be played
         """
 
-        if not isinstance(cards, list):
+        if not isinstance(card_indices, list):
             raise Exception("cards must be a list of ints")
 
-        for card in cards:
+        for card in card_indices:
             if card not in range(52):
                 raise Exception("Index must be an integer between 0 and 51 inclusive.")
 
         # Weird python indexing, but it's right
         return {
-            Players.NORTH: BridgeHand(cards[0:13]),
-            Players.EAST: BridgeHand(cards[13:26]),
-            Players.SOUTH: BridgeHand(cards[26:39]),
-            Players.WEST: BridgeHand(cards[39:52]),
-        }
\ No newline at end of file
+            Players.NORTH: Deck.__generate_bridge_hand(card_indices, 0, 13),
```

Comment:
Just looking at the logic... I think this will return 2, 3, 4, ..., K, A of Clubs.